features/page-setup.yml

```
@@ -0,0 +1,14 @@
+name: Page setup
+description: The `@page` rule sets the page-specific dimensions and margins for content such as printed documents, ebooks, or slides.
```

For consistency with other descriptions, to mention CSS.

```suggestion
description: The `@page` CSS at-rule sets the page-specific dimensions and margins for content such as printed documents, ebooks, or slides.
```